homeassistant/components/cover/mqtt.py

```diff
@@ -190,3 +254,35 @@ def async_stop_cover(self, **kwargs):
         mqtt.async_publish(
             self.hass, self._command_topic, self._payload_stop, self._qos,
             self._retain)
+
+    @asyncio.coroutine
+    def async_open_cover_tilt(self, **kwargs):
+        """Tilt the cover open."""
+        mqtt.async_publish(self.hass, self._tilt_command_topic,
+                           self._tilt_open_position, self._qos, self._retain)
+
+    @asyncio.coroutine
+    def async_close_cover_tilt(self, **kwargs):
+        """Tilt the cover closed."""
+        mqtt.async_publish(self.hass, self._tilt_command_topic,
+                           self._tilt_closed_position, self._qos, self._retain)
+
+    @property
+    def current_cover_tilt_position(self):
+        """Return current position of cover tilt."""
+        return self._tilt_value
+
+    @asyncio.coroutine
+    def async_set_cover_tilt_position(self, **kwargs):
+        """Move the cover tilt to a specific position."""
+        if ATTR_TILT_POSITION in kwargs:
+            position = float(kwargs[ATTR_TILT_POSITION])
+
+            # The position needs to be between min and max
+            tilt_range = self._tilt_max - self._tilt_min
+            percentage = position / 100.0
+            level = round(tilt_range * percentage)
+
+            _LOGGER.info("setting tilt value to " + str(level))
```

Comment:
Again, not sure if log is necessary, but if you think so, change to a format string instead of appending. Also if we want to keep it we should probably include entity_id as well.
```python
_LOGGER.info('Setting tilt value for %s to %d', self.entity_id, level)
```